airbyte-webapp/src/setupTests.tsx

```diff
@@ -4,3 +4,13 @@
 // learn more: https://github.com/testing-library/jest-dom
 import "@testing-library/jest-dom/extend-expect";
 import "@testing-library/jest-dom";
+import React from "react";
+
+// hack to fix tests. https://github.com/remarkjs/react-markdown/issues/635
+jest.mock(
+  "components/Markdown",
+  () =>
+    function ReactMarkdown({ children }: React.PropsWithChildren<unknown>) {
+      return <>{children}</>;
+    }
+);
```

Comment:
Could we use jest's [best practice](https://jestjs.io/docs/manual-mocks#mocking-user-modules) and move the mock into itself into a `__mock__/` folder beside the implementation and just activate it here for all files calling `jest.mock("components/Markdown")`?

Reply:
Good idea!